Ensure mediaURL contains a valid protocol

glitsj16 opened the pull request:
Fixes https://github.com/dicer/auto-tatort/issues/20.

Commit:
Ensure mediaURL contains a valid protocol

Fixes https://github.com/dicer/auto-tatort/issues/20.

dicer commented on autoTatort.py:
```diff
@@ -230,6 +230,10 @@ def checkForHDFile(url):
 				debug("We have only one stream. Will download it")
 
+			# ensure we always have a valid protocol
+			if not "https:" in mediaURL:
```
Maybe this should be `if not mediaURL.startswith("https:")`? Or even better: `if not mediaURL.startswith("http")`? The second one will not accidentally put https: in front of http URLs

glitsj16 replied:
Thanks for the suggestions. I agree the second one is the better way to go. Updated the PR accordingly.

Commit:
Avoid protocol mishaps for HTTP

Improved as per https://github.com/dicer/auto-tatort/pull/21#pullrequestreview-546323697.

dicer commented:
Thanks!
